packages/widgetbook/lib/src/fields/int_input_field.dart

```diff
@@ -0,0 +1,17 @@
+import 'field_codec.dart';
+import 'field_type.dart';
+import 'num_input_field.dart';
+
+class IntInputField extends NumInputField<int> {
```

Comment:
Please add a doc comment here to document as follows:

```suggestion
/// [Field] that builds [TextFormField] for [int] values.
class IntInputField extends NumInputField<int> {
```